example/settings/celery.py

```diff
@@ -0,0 +1,13 @@
+import os
+import sys
+
+from celery import Celery
+
+from django.conf import settings
+
+app = Celery('mallpay')
```

Comment:
that's an oddly specific example name :trollface:

Reply:
shiit